backend/src/main/java/feedzupzup/backend/feedback/application/event/OrganizationFeedbackCountHandler.java

```diff
@@ -0,0 +1,31 @@
+package feedzupzup.backend.feedback.application.event;
+
+import feedzupzup.backend.feedback.event.OrganizationFeedbackCountEvent;
+import feedzupzup.backend.organization.application.OrganizationStatisticService;
+import feedzupzup.backend.organization.dto.response.OrganizationStatisticResponse;
+import feedzupzup.backend.sse.service.SseService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.event.TransactionPhase;
+import org.springframework.transaction.event.TransactionalEventListener;
+
+@Component
+@RequiredArgsConstructor
+public class OrganizationFeedbackCountHandler {
+
+    private final OrganizationStatisticService organizationStatisticService;
+    private final SseService sseService;
+
+    @TransactionalEventListener(phase = TransactionPhase.AFTER_COMMIT)
+    @Async
+    public void handleFeedbackCreatedEvent(final OrganizationFeedbackCountEvent organizationFeedbackCountEvent) {
+        final OrganizationStatisticResponse organizationStatisticResponse = organizationStatisticService.getStatistic(
+                organizationFeedbackCountEvent.organizationUuid());
```

Comment:
```suggestion
                final FeedbackAmount feedbackAmount = organizationStatisticRepository.findFeedbackAmountByOrganizationId(organization.getId());
```

p2)
위에서 언급했던 이벤트를 feedback 생성 이벤트로 변경하면, 위 코드로 쉽게 변경할 수 있을 것 같아요.
기존 코드는 너무 dto에 의존하고 있네요.

Reply:
repository를 사용해 dto와 의존성을 줄이는거엔 동의합니다! 
그런데, feedback 생성 이벤트로 변경하라는 말에 대해서 이해를 못 했어요ㅜㅜ